src/components/ActivitiesListFilters.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {connect} from 'react-redux';
```

Comment:
in general... it looks ugly :P more often we write such an imports with spaces around `{` and `}`. So:
`{ connect }`. Don't fight with that. If you will setup prettier, it will handle such a stuff for you ;)